essentials/src/main/java/Lab1.java

```diff
@@ -35,31 +38,45 @@ public static void main (String[] args) {
     private static Pipeline buildPipeline() {
         Pipeline p = Pipeline.create();
 
-        // DEFINE THE DATA Directory and create an empty text file in it.
-        // New lines added to this file will be our source data
-        final String DIRECTORY = "data/";
+        // STEP 1: Print a stream to the console
 
-        // 1 - Stream new lines into the job
-        // - use drawFrom on your pipeline
-        // - Use fileWatcher source from com.hazelcast.jet.pipeline.Sources
+        StreamSource<Long> source = TestSources.itemStream(1, (ts, seq) -> seq);
```

Comment:
I would remove the lambda here, it doesn't add anything.

Reply:
The idea is to have a stream of Longs, not stream of SimpleEvents. This enables simpler switch to file watcher source in step two of the first lab